Add In-Kind sponsors blog (#376)
pythonindia/inpycon-blog: content/blog/2021/announcing-in-kind-sponsorship.md

> Title:Call for In-Kind Sponsors for PyCon India 2021

Reviewer: ```suggestion
Title: This year, PyCon India 2021 calls on In-kind sponsors. Read all about it here!
```
Too flashy? 🎃 Let's get folks excited ..

Date: 2021-06-06 17:00
Category: 2021
Slug: call-for-in-kind-sponsors

> Summary: Want to give back to the Python community and have more reach among the community members? Here's your chance to put your organization in the limelight of the Python community by sponsoring the largest Python conference in India.

Reviewer: ```suggestion
Summary: Want to give back to the Python community and reach more people among the community than ever before? Here's your chance now to put your organization in the limelight of the Python community by sponsoring the largest Python conference of this year!
```

> As you all might know that [PyCon India 2021](https://in.pycon.org/2021/) is happening online and we are not taking sponsorship for this year's event and all logistical expenses are being managed through the ticket amount collection itself. Ticket prices are also kept very minimal so that everyone can afford them. However, we are starting with In-kind Sponsorship this year so that Companies/Organizations who are still interested to contribute to the event and add value to the community can come forward and make the most out of this opportunity.

Reviewer: Might be better to break that first sentence. With the 2 `and` it's quite long

> As you all might know that [PyCon India 2021](https://in.pycon.org/2021/) is happening online and we are not taking sponsorship for this year's event and all logistical expenses are being managed through the ticket amount collection itself. Ticket prices are also kept very minimal so that everyone can afford them. However, we are starting with In-kind Sponsorship this year so that Companies/Organizations who are still interested to contribute to the event and add value to the community can come forward and make the most out of this opportunity.

Reviewer: ```suggestion
As you all might know that [PyCon India 2021](https://in.pycon.org/2021/) is happening online and we are not taking sponsorship for this year's event and all logistical expenses are being managed through the ticket amount collection itself. Ticket prices for the conference have been kept very minimal so everyone could afford them. Henceforth, this year we initiated In-kind Sponsorships at PyCon India. Companies and organizations with an interest to contribute and add value to event and the community can come forward to make the most out of this opportunity.
```

Better alternatives/suggestions for initiating needed.

Reviewer 2: what about 'started'?

> As you all might know that [PyCon India 2021](https://in.pycon.org/2021/) is happening online and we are not taking sponsorship for this year's event and all logistical expenses are being managed through the ticket amount collection itself. Ticket prices are also kept very minimal so that everyone can afford them. However, we are starting with In-kind Sponsorship this year so that Companies/Organizations who are still interested to contribute to the event and add value to the community can come forward and make the most out of this opportunity.

Reviewer: Reading this again, I think this is an important para. 
@Sc0rpi0n101 add a line about "We got a lot of queries from sponsors this year and we wanted them to still be a part of the conf. Hence this.." I remember did reach out quite a bit interested to sponsor the conf

## What are In-Kind Sponsors?

> In-kind sponsorship is a type of sponsorship where the sponsor agrees to provide swags (value-in-kind) instead of cash as part of a sponsorship agreement. These benefits/swags would be directly given to PyCon India 2021 attendees. In return for this value add/sponsorship, PyCon India 2021 team would be providing certain benefits to Sponsors, please refer to the Sponsor-Benefits section below. Regarding, what type of swags/benefits can be given to attendees, please refer to the guidelines below.

Reviewer: ```suggestion
In-kind sponsorship is a type of sponsorship where the sponsor agrees to provide swags (either physical or virtual, value-in-kind) instead of cash as part of the sponsorship agreement with PyCon India. These swags or benefits would be made directly available to all PyCon India 2021 attendees. In return for this value-added sponsorship, PyCon India 2021 team would be providing certain benefits to sponsors before, during, and after the event. 
```

Reviewer: Yo @anistark we doing physical too? 
if not @Sc0rpi0n101 disregard that bit

Reviewer: @anistark ^

Reviewer 3: Yes. Can be some goodies or books, etc that they can send directly to the participants.

Reviewer: Cool beans

## What's in it for the sponsors?

> This act of goodwill and giving back to the community can help the sponsors increase their brand exposure and elevate their business identity within the Python community. Sponsors also get to have improved interaction with the PyCon India attendees and the Python community in India. Along with all these benefits, the sponsors also get:

Reviewer: ```suggestion
This act of goodwill while giving back to the community helps sponsors increase their brand exposure and elevate their business identity within the Python community. Sponsors also get to have improved interaction with the PyCon India attendees and get a chance to be represented at the forefront of the biggest Python community in India. The sponsors will additionally be provided with:
```

Getting the tone right.

> * Logo on the website and program guide

Reviewer: ```suggestion
* Company logo on the website and PyCon India 2021 program guide.
```

> * One social media post and blog announcement

Reviewer: ```suggestion
* One social media post on all official platforms and blog announcement.
```

> * Mention in opening and closing notes during the conference (18th - 19th Sep)

Reviewer: ```suggestion
* Mention in the opening and closing notes during the conference (18th - 19th Sep)
```

> * Logo Display at the end of the talk

Reviewer: ```suggestion
* Logo display at the end of the talk.
```

> * One email to attendees

Reviewer: ```suggestion
* One email to all registered attendees.
```

> * Listing sponsors on Hopin platform

Reviewer: ```suggestion
* Sponsors listing on the PyCon India conference platform - Hopin.
```

> * One free ticket to each Sponsor

Reviewer: ```suggestion
* One free ticket to each sponsor.
```

> * Assistance from PyCon India team to send digital swags to attendees

Reviewer: ```suggestion
* Assistance from PyCon India team to send digital swags to attendees.
```

## Guidelines for Swags/Benefits

> Considering it's an online conference we'd want Sponsors to follow certain rules before deciding swags for PyCon India 2021 attendees. Below are these rules -

Reviewer: ```suggestion
With respect to this year's conference being hosted online. We want our prospective sponsors to follow certain rules before deciding on swags for PyCon India 2021 attendees. Please read the guidelines below carefully. 
```

> 1. The swag should not be a promotional item, though a valid swag can definitely contain the company's branding

Reviewer: ```suggestion
1. The swag should not be a promotional item, though a valid swag can definitely contain the company's branding.
```

> 2. The swag could be digital in nature or tangible item as well but it should be approved by the PyCon India Team

Reviewer: ```suggestion
2. The swag could be digital in nature or tangible item as well but it should be approved by the PyCon India team.
```

> 3. The sponsors are allowed to send brochures or promotional content up to a maximum of 6 pages to the attendees alongside the Swags they will be sending.

Reviewer: ```suggestion
3. The sponsors are allowed to send brochures or promotional content up to a maximum of 6 pages to the attendees alongside the swags they will be sending.
```

> 4. If the Sponsors are sending tangible swag items then logistics would have to be handled by the Sponsors themselves. PyCon India team would provide necessary assistance to the sponsors in case any information is required.

Reviewer: ```suggestion
4. If the sponsors are sending tangible swag items then logistics would have to be handled by the sponsors themselves. PyCon India team would provide necessary assistance to the sponsors if any further information is required.
```

> 5. Sponsors would be allowed to send Swags/Goodies to only those attendees who have provided their consent and shared their address for delivery

Reviewer: ```suggestion
5. Sponsors would be allowed to send swags/goodies to only those attendees who have provided their consent and shared their address for delivery.
```

> 6. Digital swags will be shared to PyCon India attendees by PyCon India 2021, Sponsors will have to forward the digital swag data and we'll take care of sending these swags

Reviewer: ```suggestion
6. Digital swags shall be shared with attendees only through PyCon India 2021 official channels. Sponsors need to forward their digital swag to the team beforehand, and the team will be sending the swags to the attendees.
```

> **_Please Note_ - At any point in time, we'll not be sharing any attendee-related data with the sponsor unless there is explicit consent provided to us by the attendees.**

Reviewer: ```suggestion
**Please Note: At any point in time, the PyCon India team will not be sharing any attendee-related information or data with the sponsor(s) without the explicit consent provided to the team by the attendees.**
```

It is not mandatory that everyone sending a proposal to become an In-kind Sponsor will be selected as a Sponsor. The PyCon India team would like to understand motivation and background and then would take a decision on selecting In-kind Sponsors.

> 
> **If you're interested in applying to be an In-kind sponsor for PyCon India 2021, please fill [this form](https://forms.gle/4tfmaAfdsHPnQ37y7)**

Reviewer: ```suggestion
```

> It is not mandatory that everyone sending a proposal to become an In-kind Sponsor will be selected as a Sponsor. The PyCon India team would like to understand motivation and background and then would take a decision on selecting In-kind Sponsors.
> 
> **If you're interested in applying to be an In-kind sponsor for PyCon India 2021, please fill [this form](https://forms.gle/4tfmaAfdsHPnQ37y7)**

Reviewer: ```suggestion
**If you're interested in applying to be an In-kind sponsor for [PyCon India 2021](https://in.pycon.org/2021/), please fill [this form](https://forms.gle/4tfmaAfdsHPnQ37y7) at the earliest.** Each application for in-kind sponsors will be reviewed and assessed by the PyCon India team. The PyCon India team would like to understand the motivation and background of each organization applying to be an in-kind sponsor and would be making a decision after the review process is complete. We look forward to all your applications and we can't wait to see what this partnership with the organizations brings to the event this year.
```

> Title: PyCon India 2021 calls on In-kind sponsors. Read all about it here!

Reviewer: How about? PyCon India 2021 opens call for ...
